src/Conversion/ONNXToTorch/ConvertONNXToTorch.cpp

```diff
@@ -89,18 +103,20 @@ struct FrontendToTorchLoweringPass
 } // end anonymous namespace.
 
 void FrontendToTorchLoweringPass::runOnOperation() {
-  ModuleOp module = getOperation();
+  // Should not run this pass if user didn't provide "--run-torch-pass" as
+  // command line option.
+  if (!RunTorchPass)
+    return;
```

Comment:
It would be more usual to handle this in `third-party/onnx-mlir/src/Compiler/CompilerUtils.cpp` to selectively add the passes to be run rather than in the passes themselves.

Reply:
Yes, you are right.  Let's keep this as a TODO item.  Is there some mechanism in this PR mechanism where we can maintain a list of TODOs?  If so, can you please do the needful to add this to that list?

I'm marking this as resolved for now.